blueprints/route/index.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-env node */
+
+var fs                   = require('fs-extra');
```

Comment:
Lines 3-7 don't follow this own style guide ;-) https://github.com/softlayer/ember-style-guide/blob/master/javascript.md#whitespace